cypress/fixtures/grantPolicy.json

```diff
@@ -0,0 +1,575 @@
+{
+    "xrn:firebolt:capability:account:id": {
+        "use": {
+            "options": [
+                {
+                    "steps": [
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:fingerprint"
+                        }
+                    ]
+                },
+                {
+                    "steps": [
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:pinchallenge",
+                            "configuration": {
+                                "pinSpace": "purchase"
+                            }
+                        },
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:acknowledgechallenge"
+                        }
+                    ]
+                }
+            ],
+            "scope": "device",
+            "lifespan": "appActive",
+            "overridable": false
+        }
+    },
+    "xrn:firebolt:capability:device:make": {
+        "use": {
+            "options": [
+                {
+                    "steps": [
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:pinchallenge",
+                            "configuration": {
+                                "pinSpace": "purchase"
+                            }
+                        },
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:acknowledgechallenge"
+                        }
+                    ]
+                },
+                {
+                    "steps": [
+                        {
+                            "capability": "xrn:firebolt:capability:usergrant:fingerprint"
```

Comment:
you need to remove "capability": "xrn:firebolt:capability:usergrant:fingerprint or need to remove the grantPolicy itself. This is not mentioned in the spec document